Core/src/org/sleuthkit/autopsy/corecomponents/DataResultViewerTable.java

```diff
@@ -81,12 +81,12 @@
 import org.sleuthkit.autopsy.coreutils.Logger;
 import org.sleuthkit.autopsy.coreutils.MessageNotifyUtil;
 import org.sleuthkit.autopsy.coreutils.ThreadConfined;
-import org.sleuthkit.autopsy.datamodel.NodeProperty;
-import org.sleuthkit.autopsy.datamodel.NodeSelectionInfo;
```

Comment:
There shouldn't be any changes to this file.

Reply:
I am fixing this problem with negative page number https://basistech.atlassian.net/browse/VIK-5395?focusedCommentId=287254&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-287254

Reply:
Yes but again that is not strictly related to this story (logical imager cancellation) and should be a separate JIRA/PR.

Reply:
ok